src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +56,40 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+}
+
+/**
+ * Takes in a Person's address in the address book by block, street, unit, postal code
+ */
+
+public interface Address{
+    public String userInput();
+}
+
+public class Block implements Address{
+    public String userInput(){
+        Scanner sc = new Sanner(System.in);
+        return sc.nextString();
+    } 
+}
+
+public class Street implements Address{
```

Comment:
need a space between the text and braces. take a look at the coding standards that we follow :-)